README.md

```diff
@@ -45,17 +46,17 @@ You can configure a route for feature-controls in your app. This will add a rout
 
 ```js
 // app/router.js
-import EmberRouter from '@ember/routing/router';
-import config from './config/environment';
-import featureControlsRouteSetup from 'ember-feature-controls/route-setup';
+import EmberRouter from "@ember/routing/router";
```

Comment:
is that normal that you get double quotes everywhere ?

Reply:
Yes it's the default configuration for eslint for an ember-project. We are not using peopledoc config here.